src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/design/ArtisticDesignForm.tsx

```diff
@@ -104,7 +104,7 @@ export const ArtisticDesignForm = () => {
                 <Table sx={{ '& th,td': { borderBottom: 1 } }}>
                   <TableHead>
                     <TableRow sx={{ '& th,td': { borderBottom: 1 } }}>
-                      <TableCell>{t('registration.resource_type.artistic.exhibition_place')}</TableCell>
+                      <TableCell>{t('registration.resource_type.artistic.place')}</TableCell>
```

Comment:
Denne nøkkelen kan fjernes, så bruker vi `common.place` som vi allerede har i stedet :) 
```suggestion
                      <TableCell>{t('common.place')}</TableCell>
```